tinygrad/runtime/ops_python.py

```diff
@@ -89,8 +89,7 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
             ul[i] = [x[2-arg[0]] for x in warp]
         elif uop is UOps.CONST:
           ul[i] = [[arg] * warp_size for _ in range(dtype.count)] if dtype.count > 1 else [arg] * warp_size
-        elif uop is UOps.DEFINE_ACC:
-          ul[i] = [[inp[0][0]] * warp_size for _ in range(dtype.count)] if dtype.count > 1 else [inp[0][0]] * warp_size
+        elif uop is UOps.DEFINE_ACC: ul[i] = inp[0] if dtype.count > 1 else [inp[0][0]] * warp_size
```

Comment:
are you sure this is correct?
uops tests failing:
https://github.com/tinygrad/tinygrad/actions/runs/9868214822/job/27282699306?pr=5322

`IMAGE=2 PYTHON=1 python3 test/test_ops.py TestOps.test_gemm`

Reply:
I'm not sure it's correct, but I think it is. I'm investigating the failure, but I haven't found many clues yet as to what's going on.

Reply:
still haven't cracked this yet, but i noticed that this test only fails with IMAGE=2. IMAGE=1 & no IMAGE envvar passes. printing and diffing the ul[i] values in this branch and master show that the values are the same. puzzling, but I'm getting closer.

Reply:
You were right about this 😄 I fixed it, but i hope i can make my fix less ugly in a cleanup patch. [0][0][0] doesn't seem that nice.

The new failures are unrelated to my latest changes. Bisecting through my git history, they started appearing after I rebased on the new lowerer. Investigating.